vividus-plugin-rest-api/src/test/java/org/vividus/http/transformer/JsonRestApiTableTransformerTests.java

```diff
@@ -74,19 +87,40 @@ void testTransformFromUrl() throws IOException
         when(httpClient.doHttpGet(UriUtils.createUri(URL_VALUE))).thenReturn(httpResponse);
         when(httpResponse.getResponseBodyAsString()).thenReturn(JSON_DATA);
         testTransform(URL_PROPERTY + EQUAL + URL_VALUE);
-        assertThat(logger.getLoggingEvents(), is(List.of(warn(
-                "`{}` parameter of `FROM_JSON` table transformer is deprecated and will be removed in VIVIDUS 0.6.0. "
-                        + "`{}` parameter must be used instead.", URL_PROPERTY, VARIABLE_PROPERTY)
+        assertThat(logger.getLoggingEvents(), is(List.of(getDeprecatedEvent(URL_PROPERTY, VARIABLE_NAME_PROPERTY)
         )));
     }
 
+    static Stream<Arguments> parameters()
+    {
+        return Stream.of(
+            Arguments.of(VARIABLE_PARAMETER,
+                is(List.of(getDeprecatedEvent(VARIABLE_PROPERTY, VARIABLE_NAME_PROPERTY)))),
+            Arguments.of(VARIABLE_NAME_PARAMETER, empty())
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("parameters")
+    void testTransformFromVariable(String parameter, Matcher expectedLogEvent)
+    {
+        when(variableContext.getVariable(VAR_NAME)).thenReturn(JSON_DATA);
+        testTransform(parameter + VAR_NAME);
+        assertThat(logger.getLoggingEvents(), expectedLogEvent);
```

Comment:
```suggestion
        assertThat(logger.getLoggingEvents(), is(expectedLogEvent));
```
and remove `is` from test arguments

Reply:
Replaced with pure jupiter assertions